### ISSUE TITLE
Generic Flair MultiLabelPool class

### ISSUE THREAD

**Issue by xkianteb** [rajcscw/nlp-gym#6]
Are you asking pool request? I would like to help maintain the repo if you are open to it. I think this project is super cool and relevant to the research that I do. Making a generic Flair MultiLabelPool class would give access to all the sequence label datasets available in the flair repo.

**Comment by rajcscw** [comment]
Yes, that is a good idea. We could also think about exposing sequence tagging datasets in hugging face datasets repo. Regarding the maintenance, sure feel free to support and contribute. I have been following your active learning research, would also like to hear in what context you would like to use nlp_gym :)
